### ISSUE TITLE
Ability to mount BOTH outer and hidden volume simultaneously

### ISSUE THREAD

**Issue by be5invis** [veracrypt/VeraCrypt#1319]
# Desired behavior

VeraCrypt could provide a functionality that when mounting an encrypted volume with hidden sub-volume, if the user provides the password for both of them, then VC could mount BOTH the other and hidden volumes as two disk drives. It will make moving data between outer and hidden volumes much simpler.

# Screenshots/Mockup/Designs

N/A. Maybe just another checkbox in the mount window.

# Additional information

There is already a write protection mechanism for outer volumes, so if we implement dual-mounting, the user should be able to operate either parts without accidentally overwriting each other.

# Your Environment

Please tell us more about your environment

**VeraCrypt version:**  1.26.10

**Operating system and version:** Windows 11 latest

**System type:** AMD64

**Comment by FearNaBoinne** [comment]
I can see the benefits of this, but there's also drawbacks, as it impacts the plausible deniability even more than simply having hidden volume protection on.
However, having hidden volume protection on in and of itself is already a bit of a risk factor if you cannot unmount before someone gains access...

**Comment by wolph** [comment]
For plausible deniability it would be useful if the outer volume got regular activity as well. If it's just an unused dummy volume it doesn't really make sense, so being able to mount both would be useful.

The point of using a hidden volume is to actually use it, right? So during it's mount the plausible deniability argument is moot anyhow. And at those times it would be useful to be able to modify both volumes simultaneously.

**Comment by FearNaBoinne** [comment]
Agreed, and I could totally see additional benefit if you accidentally saved something in the outer that needs to be in the hidden or vice versa. Now you have to copy it to another (non encrypted?) storage, unmount and remount the other volume. With dual mount, you could copy from one to the other without it ever having to be stored elsewhere.
